cli/pcluster/config/cfn_param_types.py

```diff
@@ -55,7 +55,14 @@ def from_cfn_params(self, cfn_params):
         cfn_converter = self.definition.get("cfn_param_mapping", None)
         if cfn_params:
             cfn_value = get_cfn_param(cfn_params, cfn_converter) if cfn_converter else "NONE"
-            self.value = self.get_value_from_string(cfn_value)
+            self.value = (
+                None
+                if self.key == "cluster_resource_bucket"
+                and get_cfn_param(cfn_params, "RemoveBucketOnDeletion") == "True"
+                else self.get_value_from_string(cfn_value)
+            )
+            # When the value of "RemoveBucketOnDeletion" is "True", the "cluster_resource_bucket" is not created by the
+            # user.
```

Comment:
Minor: it's a bit odd having this comment breaking like this; this line only has 1 word. Try breaking at the comma, maybe?

Reply:
Yeah! Done!